{
    "driver_name" : "icl",
,
        {
            "diagnostic_num" : "10120",
            "type" : "command line warning",
            "msg" : "overriding '\/O3' with '\/Ox'"
        },
        {
            "diagnostic_num" : "10120",
            "type" : "command line warning",
            "msg" : "overriding '\/Ox' with '\/O3'"
        },
        {
            "diagnostic_num" : "10121",
            "type" : "command line warning",
            "msg" : "overriding '\/Qipo1' with '\/Qipo'"
        },
        {
            "diagnostic_num" : "10121",
            "type" : "command line warning",
            "msg" : "overriding '\/fp:fast=2' with '\/fp:strict'"
        },
        {
            "diagnostic_num" : "10121",
            "type" : "command line warning",
            "msg" : "overriding '\/Qipo' with '\/Qipo1'"
        }
    ],
    "options" : [ "/Ot", "/Oi", "/QxHost", "/EHc", "/Qcf-protection:none", "/Qunroll-aggressive", "/Qopt-multi-version-aggressive", "/Qvec-guard-write", "/Qopt-subscript-in-range", "/Qguide-file", "/Qopt-mem-layout-trans", "/Qalign-loops", "/Qprof-src-dir", "/Qprof-file:profiling_summary", "/Qpar-schedule-auto", "/Qpar", "/Qpar-loops:new", "/fp:fast=2", "/Qfinite-math-only", "/Ob2", "/Qinline-min-size-", "/Qemit-build-options", "/Qemit-messages", "/Zc:__cplusplus", "/Zc:inline", "/Zc:rvalueCast", "/Zc:strictStrings", "/MP", "/GS-", "/GF", "/Qfnalign", "/Qms2", "/Qvc14.2", "/D_HAS_CONDITIONAL_EXPLICIT=0", "/Qipo1", "/O3", "/arch:AVX2", "/Ox", "/GS-", "/O3", "/c", "/Qm64", "/IZ:\cpython27\externals\sqlite\", "/IZ:\cpython27\Include", "/IZ:\cpython27\PC", "/Zi", "/nologo", "/W3", "/O3", "/Ob2", "/Oi", "/Ot", "/Qipo", "/Qparallel", "/Quse-intel-optimized-headers", "/D:__INTEL_COMPILER=1910", "/D:MODULE_NAME="sqlite3"", "/D:WIN32", "/D:_WIN64", "/D:_M_X64", "/D:NDEBUG", "/D:Py_BUILD_CORE_MODULE", "/D:_WINDLL", "/GF", "/MD", "/GS-", "/Gy", "/arch:CORE-AVX2", "/fp:strict", "/QxHost", "/Zc:wchar_t", "/Zc:forScope", "/FoZ:\cpython27\PCbuild\obj\amd64_Release\_sqlite3\", "/FdZ:\cpython27\PCbuild\obj\amd64_Release\_sqlite3\vc142.pdb", "/TC", "/FC", "/Qipo1", "/GS-", "/O3" ],
    "objects" : [
        {
            "object_name" : "Z:\\cpython27\\PCbuild\\obj\\amd64_Release\\_sqlite3\\connection.obj",
            "source_name" : "..\\Modules\\_sqlite\\connection.c",
            "start_time" : "2019-12-27T01:21:11Z",
            "end_time" : "2019-12-27T01:21:11Z"
        }
        ,
        {
            "object_name" : "Z:\\cpython27\\PCbuild\\obj\\amd64_Release\\_sqlite3\\cursor.obj",
            "source_name" : "..\\Modules\\_sqlite\\cursor.c",
            "start_time" : "2019-12-27T01:21:11Z",
            "messages" : [
                {
                    "msg" : "catastrophic error: cannot open source file \"sqlite3.h\"\n  #include \"sqlite3.h\"\n                      ^",
                    "file" : "Z",
                    "type" : "\\cpython27\\Modules\\_sqlite\\connection.h(33)"
                }
            ],
            "end_time" : "2019-12-27T01:21:12Z"
        }
        ,
        {
            "object_name" : "Z:\\cpython27\\PCbuild\\obj\\amd64_Release\\_sqlite3\\connection.obj",
            "source_name" : "..\\Modules\\_sqlite\\connection.c",
            "start_time" : "2019-12-27T01:21:12Z",
            "end_time" : "2019-12-27T01:21:12Z"
        }
        ,
        {
            "object_name" : "Z:\\cpython27\\PCbuild\\obj\\amd64_Release\\_sqlite3\\microprotocols.obj",
            "source_name" : "..\\Modules\\_sqlite\\microprotocols.c",
            "start_time" : "2019-12-27T01:21:12Z",
            "messages" : [
                {
                    "msg" : "catastrophic error: cannot open source file \"sqlite3.h\"\n  #include \"sqlite3.h\"\n                      ^",
                    "file" : "Z",
                    "type" : "\\cpython27\\Modules\\_sqlite\\connection.h(33)"
                }
            ],
            "end_time" : "2019-12-27T01:21:12Z"
        }
        ,
        {
            "object_name" : "Z:\\cpython27\\PCbuild\\obj\\amd64_Release\\_sqlite3\\connection.obj",
            "source_name" : "..\\Modules\\_sqlite\\connection.c",
            "start_time" : "2019-12-27T01:21:12Z",
            "end_time" : "2019-12-27T01:21:12Z"
        }
        ,
        {
            "object_name" : "Z:\\cpython27\\PCbuild\\obj\\amd64_Release\\_sqlite3\\module.obj",
            "source_name" : "..\\Modules\\_sqlite\\module.c",
            "start_time" : "2019-12-27T01:21:12Z",
            "messages" : [
                {
                    "msg" : "catastrophic error: cannot open source file \"sqlite3.h\"\n  #include \"sqlite3.h\"\n                      ^",
                    "file" : "Z",
                    "type" : "\\cpython27\\Modules\\_sqlite\\connection.h(33)"
                }
            ],
            "end_time" : "2019-12-27T01:21:12Z"
        }
        ,
        {
            "object_name" : "Z:\\cpython27\\PCbuild\\obj\\amd64_Release\\_sqlite3\\connection.obj",
            "source_name" : "..\\Modules\\_sqlite\\connection.c",
            "start_time" : "2019-12-27T01:21:12Z",
            "end_time" : "2019-12-27T01:21:12Z"
        }
        ,
        {
            "object_name" : "Z:\\cpython27\\PCbuild\\obj\\amd64_Release\\_sqlite3\\statement.obj",
            "source_name" : "..\\Modules\\_sqlite\\statement.c",
            "start_time" : "2019-12-27T01:21:12Z",
            "messages" : [
                {
                    "msg" : "catastrophic error: cannot open source file \"sqlite3.h\"\n  #include \"sqlite3.h\"\n                      ^",
                    "file" : "Z",
                    "type" : "\\cpython27\\Modules\\_sqlite\\connection.h(33)"
                }
            ],
            "end_time" : "2019-12-27T01:21:12Z"
        }
        ,
        {
            "object_name" : "Z:\\cpython27\\PCbuild\\obj\\amd64_Release\\_sqlite3\\connection.obj",
            "source_name" : "..\\Modules\\_sqlite\\connection.c",
            "start_time" : "2019-12-27T01:21:12Z",
            "end_time" : "2019-12-27T01:21:12Z"
        }
        ,
        {
            "object_name" : "Z:\\cpython27\\PCbuild\\obj\\amd64_Release\\_sqlite3\\row.obj",
            "source_name" : "..\\Modules\\_sqlite\\row.c",
            "start_time" : "2019-12-27T01:21:12Z",
            "messages" : [
                {
                    "msg" : "catastrophic error: cannot open source file \"sqlite3.h\"\n  #include \"sqlite3.h\"\n                      ^",
                    "file" : "Z",
                    "type" : "\\cpython27\\Modules\\_sqlite\\connection.h(33)"
                }
            ],
            "end_time" : "2019-12-27T01:21:12Z"
        }
        ,
        {
            "object_name" : "Z:\\cpython27\\PCbuild\\obj\\amd64_Release\\_sqlite3\\connection.obj",
            "source_name" : "..\\Modules\\_sqlite\\connection.c",
            "start_time" : "2019-12-27T01:21:12Z",
            "end_time" : "2019-12-27T01:21:12Z"
        }
        ,
        {
            "object_name" : "Z:\\cpython27\\PCbuild\\obj\\amd64_Release\\_sqlite3\\util.obj",
            "source_name" : "..\\Modules\\_sqlite\\util.c",
            "start_time" : "2019-12-27T01:21:12Z",
            "messages" : [
                {
                    "msg" : "catastrophic error: cannot open source file \"sqlite3.h\"\n  #include \"sqlite3.h\"\n                      ^",
                    "file" : "Z",
                    "type" : "\\cpython27\\Modules\\_sqlite\\connection.h(33)"
                }
            ],
            "end_time" : "2019-12-27T01:21:12Z"
        }
        ,
        {
            "object_name" : "Z:\\cpython27\\PCbuild\\obj\\amd64_Release\\_sqlite3\\connection.obj",
            "source_name" : "..\\Modules\\_sqlite\\connection.c",
            "start_time" : "2019-12-27T01:21:12Z",
            "end_time" : "2019-12-27T01:21:12Z"
        }
        ,
        {
            "object_name" : "Z:\\cpython27\\PCbuild\\obj\\amd64_Release\\_sqlite3\\sqlite3.obj",
            "source_name" : "Z:\\cpython27\\externals\\sqlite\\\\bld\\sqlite3.c",
            "start_time" : "2019-12-27T01:21:12Z",
            "end_time" : "2019-12-27T01:21:15Z"
        }
        ,
        {
            "object_name" : "Z:\\cpython27\\PCbuild\\obj\\amd64_Release\\_sqlite3\\connection.obj",
            "source_name" : "..\\Modules\\_sqlite\\connection.c",
            "start_time" : "2019-12-27T01:21:11Z",
            "messages" : [
                {
                    "msg" : "catastrophic error: cannot open source file \"sqlite3.h\"\n  #include \"sqlite3.h\"\n                      ^",
                    "file" : "Z",
                    "type" : "\\cpython27\\Modules\\_sqlite\\connection.h(33)"
                }
            ],
            "end_time" : "2019-12-27T01:21:15Z"
        }
    ]
}
